### PR TITLE
strategy update 15. Jan 2021

### PR DESCRIPTION
**Pull request by BenWhiteJam**

### CODE REVIEW

File: docs/build-parachains-rococo.md [hunk 1]
```diff
@@ -141,6 +141,21 @@ process are the following:
...
 
+*Update: 15. Jan. 2021*
+
+- If you are considered to be included, the Rococo team will get in touch with you through the Element handle provided in the sign-up form, a rough estimate of going forward will be (in no certain order within the batch and please be assure of 5., 6., 7.):
+  - first batch: Kilt, Interlay (next week)
+  - second batch: Darwinia, Phala, Crust, HydraDX
+  - third batch: Bifrost, Starks Network, Clover, Zenlink
+  - fourth batch: ChainX, Robonomics, Patract Hub, MathWallet
+  - fith batch: we will communicate timely
+- You will have 2 days to get everything up and running after the Rococo team let's you know
```
Comment (Swader) [inline docs/build-parachains-rococo.md]:
```suggestion
- You will have 2 days to get everything up and running after the Rococo team lets you know
```

File: docs/build-parachains-rococo.md [hunk 2]
```diff
@@ -141,6 +141,21 @@ process are the following:
...
+- Make sure you build, run and test a local setup based on the `rococo-v1` branch for a while (a proof will be asked for at time)
```
Comment (Swader) [inline docs/build-parachains-rococo.md]:
```suggestion
- Make sure you build, run, and test a local setup based on the `rococo-v1` branch for a while (you will be asked for proof)
```

File: docs/build-parachains-rococo.md [hunk 3]
```diff
@@ -141,6 +141,21 @@ process are the following:
 2. Maintain at least one parachain collator
 3. Sign-up through the [Rococo V1 Parachain Registration](https://forms.gle/Eacp7RaRm3VNion16) form
 
+*Update: 15. Jan. 2021*
+
+- If you are considered to be included, the Rococo team will get in touch with you through the Element handle provided in the sign-up form, a rough estimate of going forward will be (in no certain order within the batch and please be assure of 5., 6., 7.):
```
Comment (Swader) [inline docs/build-parachains-rococo.md]: "be assure of" sounds wrong, and also the items below are not numbered, maybe number them to match this?

File: docs/build-parachains-rococo.md [hunk 4]
```diff
@@ -141,6 +141,21 @@ process are the following:
...
+  - first batch: Kilt, Interlay (next week)
```
Comment (Swader) [inline docs/build-parachains-rococo.md]: Let's put specific dates instead of "next week", for posterity

File: docs/build-parachains-rococo.md [hunk 5]
```diff
@@ -141,6 +141,21 @@ process are the following:
...
+  - second batch: Darwinia, Phala, Crust, HydraDX
+  - third batch: Bifrost, Starks Network, Clover, Zenlink
+  - fourth batch: ChainX, Robonomics, Patract Hub, MathWallet
+  - fith batch: we will communicate timely
+- You will have 2 days to get everything up and running after the Rococo team let's you know
+- Make sure you build, run and test a local setup based on the `rococo-v1` branch for a while (a proof will be asked for at time)
+- Together with the Rococo team this thing will get deployed
+- Don't worry, if you registered your slot is secured
```
Comment (Swader) [inline docs/build-parachains-rococo.md]:
```suggestion
- Don't worry, if you registered, your slot is secured
```

File: docs/build-parachains-rococo.md [hunk 6]
```diff
@@ -141,6 +141,21 @@ process are the following:
...
+
+- If you are considered to be included, the Rococo team will get in touch with you through the Element handle provided in the sign-up form, a rough estimate of going forward will be (in no certain order within the batch and please be assure of 5., 6., 7.):
+  - first batch: Kilt, Interlay (next week)
+  - second batch: Darwinia, Phala, Crust, HydraDX
+  - third batch: Bifrost, Starks Network, Clover, Zenlink
+  - fourth batch: ChainX, Robonomics, Patract Hub, MathWallet
+  - fith batch: we will communicate timely
+- You will have 2 days to get everything up and running after the Rococo team let's you know
+- Make sure you build, run and test a local setup based on the `rococo-v1` branch for a while (a proof will be asked for at time)
+- Together with the Rococo team this thing will get deployed
```
Comment (Swader) [inline docs/build-parachains-rococo.md]:
```suggestion
- The Rococo team will help you get your chain deployed
```

**Review by Swader — CHANGES_REQUESTED** [review]

**Comment by Swader** [comment]
@w3fbot prettify please

**Review by Swader — APPROVED** [review]

**Review by rphmeier — APPROVED** [review]

**Pull request merged**